src/sentry/integrations/jira/integration.py

```diff
@@ -147,6 +149,19 @@ class JiraIntegration(IssueSyncIntegration):
     resolution_strategy_key = "resolution_strategy"
     comment_key = "sync_comments"
 
+    def raise_error(self, exc: Exception, identity: Identity | None = None) -> NoReturn:
+        if isinstance(exc, ApiInvalidRequestError):
+            if exc.json:
+                error_fields = self.error_fields_from_json(exc.json)
+                if error_fields is not None:
+                    raise IntegrationFormError(error_fields).with_traceback(sys.exc_info()[2])
```

Comment:
er could we get a comment with what this does and maybe a test what the error will look like?